### PR TITLE
Feature#15 controller views home pages

### LINKED ISSUE #15: Create Controllers and Views for Home Page
# REQUIREMENTS
Create a controller to manage routes that do not require the student or the teacher to log in.
Create the view for the main page. In this view, the application must be described, that is, what it is about, what it is for, etc.
Create the view to choose type of user record. Keep in mind that you can only register as a teacher or as a student.

# Acceptance Criteria
- [ ] Controller Pages
- [ ] View for Home Page
- [ ] View to select record type

### PR DESCRIPTION
**Pull request by RobertoCarMed**
# Description
- The controller for the pages with static content was created.
- The home page contains information about the application 
- The register page allows you to select the role that you are going to fulfill within the application.

resolve #15

### CODE REVIEW

File: app/assets/stylesheets/pages.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Pages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (YilKanda) [inline app/assets/stylesheets/pages.scss:3]: If you don't occupy it, delete it.

File: app/helpers/pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module PagesHelper
+end
```
Comment (YilKanda) [inline app/helpers/pages_helper.rb:2]: If you don't occupy it, delete it.

File: app/views/pages/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+          reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+        </p>
+      </div>
+    </div>
+    
+    <div class="images">
+      <img src="mdn-logo-sm.png" alt="IMG Value Proposition">
+    </div>
+  </section>
+</div>
```
Comment (YilKanda) [inline app/views/pages/index.html.erb:79]: Add a blank line.

File: app/views/pages/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+      <h1>Lets Learn</h1>
+    </div>
+
+    <div class="description">
+      <p> 
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+        Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
+        nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in 
+        reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
```
Comment (YilKanda) [inline app/views/pages/index.html.erb:13]: It would be good if you could post some content related to your project.

**Review by YilKanda — CHANGES_REQUESTED** [review]

**Review by xFerchoVJ — DISMISSED** [review]

**Comment by brayandiazc** [comment]
Solve the points that Yil tells you, check the conflicts that the code is generating, which you can edit from github

**Comment by brayandiazc** [comment]
Remember, before uploading any changes to github first download the most current version of the project

File: app/assets/stylesheets/pages.scss (hunk 1 above)
Comment (RobertoCarMed) [reply app/assets/stylesheets/pages.scss]: Comments we are delete

File: app/views/pages/index.html.erb (hunk 3 above)
Comment (RobertoCarMed) [reply app/views/pages/index.html.erb:67]: resolved

File: app/helpers/pages_helper.rb (hunk 2 above)
Comment (RobertoCarMed) [reply app/helpers/pages_helper.rb:2]: We may need it in the future

File: app/views/pages/index.html.erb (hunk 4 above)
Comment (RobertoCarMed) [reply app/views/pages/index.html.erb]: Resolved

**Review by xFerchoVJ — APPROVED** [review]
LGTM!

**Review by brayandiazc — APPROVED** [review]

**Review by SaraNavarro — APPROVED** [review]

**Pull request merged**